oracle/provider/osmosisv2.go

```diff
@@ -30,14 +30,13 @@ type (
 	//
 	// REF: https://github.com/ojo-network/osmosis-api
 	OsmosisV2Provider struct {
```

Comment:
What do you think about changing this to `osmosis` instead of `osmosisv2` now that the other one is getting removed?

Reply:
I love the idea. What do you think about making that its own PR since this one is already large and that will add a ton more changes?

Reply:
https://github.com/ojo-network/price-feeder/issues/169